README.md

```diff
@@ -301,6 +301,16 @@ The below steps will populate your local Conan repository with dependencies for
 1. Download CityEngine SDK: `conan create -pr conan/profiles/windows-v143 conan/cesdk cesdk/3.2.10650@esri-rd-zurich/stable`
 1. Extract and package the HDK from your local Houdini installation (adjust Z to your Houdini version): `conan create -pr conan/profiles/windows-v143 conan/houdini houdini/20.0.Z@sidefx/stable` (Note: use the option `-e HOUDINI_INSTALL=C:/path/to/your/houdini/installation`, if Houdini is not installed at the standard location for Windows).
 
+##### Docker Toolchain Images
+
+This repo includes Dockerfiles for Linux/Windows toolchain images. Here is how to build those:
+- Linux: `docker build -t palladio-tc -f docker/linux/Dockerfile --build-context houdini=<path to Houdini installation> .`
```

Comment:
Known issue: add missing `--build-arg HDK_VERSION=X.Y.Z` and rename `HDK_VERSION` to `HOUDINI_VERSION`